src/app/services/player.service.js

```diff
@@ -191,6 +194,23 @@ class PlayerService {
     this.socketService.emit('volume', volume);
   }
 
+  get sliderVolume(){
+    // console.log("slider get");
+    if(Date.now() - this.volumeSliderLastUpdateTime > 1000){
```

Comment:
It's better to expose to the UI the prop `volume` and not to bind the var name to the UI component that will consume that variable.
So rename the vars to something similar to:
volumeSliderLastUpdateTime -> lastVolumeUpdateTime
volumeSliderValue -> volume
this.volume -> lastVolumeValue